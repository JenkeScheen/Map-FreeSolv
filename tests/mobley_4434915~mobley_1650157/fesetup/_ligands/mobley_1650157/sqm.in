Run semi-empirical minimization
 &qmmm
  qm_theory='AM1',grms_tol=0.0002,tight_p_conv=1,
  scfconv=1.0d-10,itrmax=500,pseudo_diag=1,
  maxcyc=1000,
  qmcharge=0,
 /
   6    C1       -0.4650       -0.0650        0.9890 
   6    C2        0.5610       -0.3990        2.0720 
   6    C3        1.6270        0.6910        2.1570 
   6    C4        1.2040       -1.7600        1.8120 
   8    O5       -0.1080       -0.4450        3.3330 
   1    H6       -1.2320       -0.8450        0.9250 
   1    H7       -0.9580        0.8930        1.1830 
   1    H8        0.0100       -0.0160        0.0030 
   1    H9        2.3650        0.4550        2.9320 
   1   H10        1.1820        1.6520        2.4370 
   1   H11        2.1530        0.8170        1.2050 
   1   H12        1.9390       -2.0000        2.5880 
   1   H13        0.4570       -2.5600        1.7840 
   1   H14        1.7480       -1.7590        0.8600 
   1   H15       -0.7830       -1.1370        3.2590 

