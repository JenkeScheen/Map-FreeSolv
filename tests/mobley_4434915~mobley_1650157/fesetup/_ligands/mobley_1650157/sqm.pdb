ATOM      1  C1  LIG     1      -0.459  -0.077   0.986  1.00  0.00           C
ATOM      2  C2  LIG     1       0.560  -0.405   2.071  1.00  0.00           C
ATOM      3  C3  LIG     1       1.619   0.687   2.158  1.00  0.00           C
ATOM      4  C4  LIG     1       1.207  -1.757   1.795  1.00  0.00           C
ATOM      5  O5  LIG     1      -0.079  -0.422   3.349  1.00  0.00           O
ATOM      6  H6  LIG     1      -1.229  -0.882   0.920  1.00  0.00           H
ATOM      7  H7  LIG     1      -0.970   0.886   1.223  1.00  0.00           H
ATOM      8  H8  LIG     1       0.051   0.015  -0.002  1.00  0.00           H
ATOM      9  H9  LIG     1       2.344   0.450   2.972  1.00  0.00           H
ATOM     10  H10 LIG     1       1.137   1.667   2.386  1.00  0.00           H
ATOM     11  H11 LIG     1       2.163   0.759   1.188  1.00  0.00           H
ATOM     12  H12 LIG     1       1.909  -2.018   2.622  1.00  0.00           H
ATOM     13  H13 LIG     1       0.427  -2.553   1.725  1.00  0.00           H
ATOM     14  H14 LIG     1       1.772  -1.721   0.835  1.00  0.00           H
ATOM     15  H15 LIG     1      -0.754  -1.109   3.312  1.00  0.00           H
