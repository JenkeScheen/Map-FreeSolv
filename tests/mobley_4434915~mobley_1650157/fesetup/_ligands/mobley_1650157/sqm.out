            --------------------------------------------------------
                             AMBER SQM VERSION 14
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 16

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.1D-09
| QMMM: Maximum number of SCF cycles :      500

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       30
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.004125 seconds
| QMMM:                 Dspev diag routine = 0.003329 seconds
| QMMM:                Dspevd diag routine = 0.002810 seconds
| QMMM:                Dspevx diag routine = 0.011266 seconds
| QMMM:                 Dsyev diag routine = 0.002846 seconds
| QMMM:                Dsyevd diag routine = 0.001968 seconds
| QMMM:                Dsyevr diag routine = 0.002050 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.000410 seconds
| QMMM:
| QMMM: Using dsyevd routine (diag_routine=6).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       -0.4650   -0.0650    0.9890
  QMMM:     2        2      C        0.5610   -0.3990    2.0720
  QMMM:     3        3      C        1.6270    0.6910    2.1570
  QMMM:     4        4      C        1.2040   -1.7600    1.8120
  QMMM:     5        5      O       -0.1080   -0.4450    3.3330
  QMMM:     6        6      H       -1.2320   -0.8450    0.9250
  QMMM:     7        7      H       -0.9580    0.8930    1.1830
  QMMM:     8        8      H        0.0100   -0.0160    0.0030
  QMMM:     9        9      H        2.3650    0.4550    2.9320
  QMMM:    10       10      H        1.1820    1.6520    2.4370
  QMMM:    11       11      H        2.1530    0.8170    1.2050
  QMMM:    12       12      H        1.9390   -2.0000    2.5880
  QMMM:    13       13      H        0.4570   -2.5600    1.7840
  QMMM:    14       14      H        1.7480   -1.7590    0.8600
  QMMM:    15       15      H       -0.7830   -1.1370    3.2590

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10      -71.7057 kcal/mol        0.0124 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -39.2082  -33.3528  -28.0553  -28.0462  -21.0607  -17.5255  -16.0640  -15.5504
 -14.2747  -13.4656  -13.0168  -12.7173  -12.5820  -12.5041  -11.7690  -10.9902
   3.4390    3.7400    3.8423    4.0494    4.3666    4.4672    4.5878    4.7272
   4.7722    4.8629    5.0155    5.1304    5.1551    6.3935

 Heat of formation   =        -71.70574605 kcal/mol  (       -3.10939448 eV)

 Total SCF energy    =     -22398.07790897 kcal/mol  (     -971.25354100 eV)
 Electronic energy   =     -85508.93980539 kcal/mol  (    -3707.94587422 eV)
 Core-core repulsion =      63110.86189642 kcal/mol  (     2736.69233322 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.240
     2      C                  0.074
     3      C                 -0.201
     4      C                 -0.240
     5      O                 -0.326
     6      H                  0.072
     7      H                  0.089
     8      H                  0.083
     9      H                  0.087
    10      H                  0.087
    11      H                  0.076
    12      H                  0.089
    13      H                  0.072
    14      H                  0.083
    15      H                  0.196
 Total Mulliken Charge =       0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -0.095   -0.777   -1.416    1.618
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       -0.4586   -0.0769    0.9857
  QMMM:     2        2      C        0.5602   -0.4046    2.0706
  QMMM:     3        3      C        1.6192    0.6874    2.1581
  QMMM:     4        4      C        1.2067   -1.7567    1.7949
  QMMM:     5        5      O       -0.0789   -0.4223    3.3489
  QMMM:     6        6      H       -1.2286   -0.8821    0.9202
  QMMM:     7        7      H       -0.9697    0.8864    1.2230
  QMMM:     8        8      H        0.0512    0.0149   -0.0017
  QMMM:     9        9      H        2.3441    0.4500    2.9723
  QMMM:    10       10      H        1.1372    1.6675    2.3858
  QMMM:    11       11      H        2.1629    0.7588    1.1875
  QMMM:    12       12      H        1.9091   -2.0176    2.6219
  QMMM:    13       13      H        0.4274   -2.5526    1.7249
  QMMM:    14       14      H        1.7720   -1.7209    0.8345
  QMMM:    15       15      H       -0.7541   -1.1092    3.3125

           --------- Calculation Completed ----------

